### PR TITLE
Graphics GDL module PR #1

### PR DESCRIPTION
**Pull request by imadejski**
This is the first pull request for the GDL module, which read in a json file to set the graphics information for the game. In this pull request, we designed the module and created the read_gdl.h and read_gdl.c which outline the desired functions and implement the new, init, and free functions for the module. We also created tests for the camera and display_dimensions structs that exist within the read_gdl files. The tests build successfully and pass.

### CODE REVIEW

**Review by dhirpalshah — APPROVED** [review]
looks good to me!

File: graphics_sandbox/gdl/read_gdl.h [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+<<<<<<< HEAD
+ * - an uninitialized camera structure
+ * - an integer for camera width
+ * - an integer for camera height
+ *
+ * Returns:
+ * - an integer value for success
+=======
+ * - an uninitialized
+>>>>>>> d26175b898a004f623b54b54f8c67cdae0ce0265
```
Comment (philipchenaux) [inline graphics_sandbox/gdl/read_gdl.h]: Please clean up this header comment.

File: graphics_sandbox/gdl/read_gdl.c [hunk 2]
```diff
@@ -0,0 +1,260 @@
+#include "read_gdl.h"
+#include "raylib.h"
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+
+graphics_t* new_graphics(display_dimensions_t *dimensions, camera_t *camera,
```
Comment (philipchenaux) [inline graphics_sandbox/gdl/read_gdl.c:8]: Make sure to add "see read_gdl.h" as a header comment for each of these functions.

**Review by philipchenaux — CHANGES_REQUESTED** [review]
Minor revisions need to be made.

**Comment by imadejski** [comment]
Minor revisions completed. See read_gdl.h header comment added and formatting of specified header comment cleaned up. No other changes requested.

**Review by philipchenaux — APPROVED** [review]
Looks good to merge!

**Comment by philipchenaux** [comment]
Issue Score: Excellent

Comments:
Great work, but please make sure that you do a thorough code review and, in the future, make sure to look out for the kind of issues the senior developer pointed out.

**Pull request merged**